Content.Server/GameObjects/Components/Interactable/Tools/CrowbarComponent.cs

```diff
@@ -18,6 +19,8 @@ public class CrowbarComponent : ToolComponent, IAfterAttack
         [Dependency] private readonly IMapManager _mapManager;
 #pragma warning restore 649
 
+        private double MAX_DISTANCE = 1.5;
```

Comment:
You should instead use the `InteractionSystem.InteractionRange` constant to do this distance check. Then it's consistent with other item interactions.